src/abstract/lib/types/primitive.ts

```diff
@@ -16,3 +16,5 @@ export type AppletWithVersion = {
 };
 
 export type NoParamVoidFunction = () => void;
+
+export type TimerCallbackFeedback = 'no-feedback' | 'request-for-postpone';
```

Comment:
Not sure what it means. Request for postponement of what?

Reply:
Yes, callback implicitly asks timer to wait for a while and repead callback again.

Reply:
Maybe, it would be a good idea to call `postpone` operation when it is needed instead of relying on strings

Reply:
reverted